### ISSUE TITLE
Final review

### ISSUE THREAD

**Issue by win13676** [Jz2249/ORIE5741#10]
The team's goal is to use explore the feasibility of using machine learning models to help financial institutions to make decisions on mortgages application more precisely and fairer. The team used the FFIEC Home Mortgage Disclosure Act of 2020 California mortgage dataset from 21ST MORTGAGE CORPORATION. The team concluded that gradient boosted decision trees are the best model for the task with the best score. The team also discovered unfairness in the model and recommend training the model with all features and masking non-financial features with default values in prediction.

Having an accurate and fair mortgage decision is important because it would avoid mortgage default and give everyone equal and fair opportunity to owning real estate and improving their livelihood. The team data processing and exploratory data analysis make sense. They impute missing data, remove unnecessary data and one hot encode the categorical feature. The usage of square and bucket features are also interesting. I'm surprise it doesn't overfit the data. The team also evaluated several models using cross validation and also have a separate test set. 

The graph in the report could be improved. The team should label the legends with the actual category instead of number. I can't tell what the number correspond to.  The box plots are hard to interpret because there're a lot of outliers and the box of the action taken vs not taken are similar to each other. The plot doesn't clearly support the team's conclusion because the median and the quantile of the plots are very similar to each other. Furthermore, the graph of the non-financial features are also hard to interpret. The graph isn't normalized by the population size of the group, so it's hard to tell which group has higher approval rate to evaluate the unfairness. Table format with percentage would be easier to compare. The team also just recommended approach to reducing unfairness but doesn't give any concrete data or fairness metrics to show that the recommendation actually work.
